compiler/src/dotty/tools/dotc/core/NameOps.scala

```diff
@@ -78,9 +78,22 @@ object NameOps {
     def isUnapplyName: Boolean = name == nme.unapply || name == nme.unapplySeq
     def isRightAssocOperatorName: Boolean = name.lastPart.last == ':'
 
-    def isOperatorName: Boolean = name match
-      case name: SimpleName => name.exists(isOperatorPart)
-      case _ => false
+    /** Does this name match `[{letter | digit} '_'] op`?
```

Comment:
Note: `letter`, `digit` and `op` are defined in https://scala-lang.org/files/archive/spec/3.4/01-lexical-syntax.html.